### PR TITLE
Updating dependencies

### PR DESCRIPTION
**Pull request by IlyaKislitsin**
## Description

Update minimal required PHP version to PHP 8.0. Added support for the Laravel 10 version.

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code _(if tests is required for my changes)_
- [x] I have made changes in `CHANGELOG.md` file

### Added

- Support Laravel `10.x`

### Changed

- Minimal require PHP version now is `8.0`
- Minimal Laravel version is `9.1`
- Composer version up to `2.5.5`

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -13,27 +13,18 @@ on:
 jobs: # Docs: <https://git.io/JvxXE>
   php:
     name: PHP ${{ matrix.php }}, ${{ matrix.setup }} setup, laravel ${{ matrix.laravel }}
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-stable
```
Comment (eldario) [inline .github/workflows/tests.yml:16]:
```suggestion
    runs-on: ubuntu-latest
```

Comment (eldario) [inline .github/workflows/tests.yml:16]:
```suggestion
    runs-on: ubuntu-latest
```

**Review by eldario — COMMENTED** [review]
wrong tag

File: .github/workflows/tests.yml (hunk 1 above)
Comment (IlyaKislitsin) [reply .github/workflows/tests.yml]: Thanks for review. I fixed the wrong tag.

**Review by eldario — APPROVED** [review]

**Comment by eldario** [comment]
Thanx for your PR!

**Pull request merged**